lib/routes/questn/community.ts

```diff
@@ -0,0 +1,76 @@
+import { Route } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import ofetch from '@/utils/ofetch';
+
+export const route: Route = {
+    path: '/community/:community_url',
+    name: 'QuestN Community Events',
+    url: 'app.questn.com',
+    maintainers: ['cxheng315'],
+    example: '/questn/community/gmnetwork',
+    parameters: {
+        community_url: 'Community URL',
+    },
+    categories: ['other'],
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['app.questn.com/:community_url'],
+            target: '/community/:community_url',
+        },
+    ],
+    handler,
+};
+
+async function handler(ctx) {
+    const url = 'https://api.questn.com/consumer/explore/entity_list/';
+
+    const params = {
+        count: ctx.req.query('limit') || '20',
+        page: '1',
+        community_url: ctx.req.param('community_url') || 'questn', // default to questn
```

Comment:
[`camelCase`](https://docs.rsshub.app/joinus/advanced/script-standard#naming)

Reply:
this is to fit target api parameter
`https://api.questn.com/consumer/explore/entity_list/?count=20&page=1&community_url=gmnetwork`
so line 41 `${url}?${new URLSearchParams(params)}` would get the correct parameter

Reply:
I'm talking about the path parameter
https://github.com/DIYgod/RSSHub/blob/2d9cdff0fb5029dbd4c758161fb9f6d243f78ef2/lib/routes/questn/community.ts#L6

Reply:
changed, thank you!